webrtc/RTCPeerConnection-createDataChannel.html

```diff
@@ -6,19 +6,56 @@
 <script>
```

Comment:
Above this line, change the title to "RTCPeerConnection.prototype.createDataChannel"